lib/src/signed_video_h26x_auth.c

```diff
@@ -332,7 +425,7 @@ set_validation_status_of_items_used_in_gop_hash(h26x_nalu_list_t *nalu_list, cha
   // Loop through the |nalu_list| and set the |validation_status| if the item is |used_in_gop_hash|
   h26x_nalu_list_item_t *item = nalu_list->first_item;
   while (item) {
-    if (item->used_in_gop_hash) {
+    if (item->used_in_gop_hash && item->validation_status == 'P') {
```

Comment:
Now you have changed the behavior compared to the description. You're setting validation status of PENDING items used in gop hash. First question, is it necessary to add the "pending" requirement? And secondly, if so you should update the description and probably the function name as well.